data/json/items/resources/wood.json

```
@@ -93,6 +93,35 @@
     "qualities": [ [ "HAMMER", 1 ] ],
     "melee_damage": { "bash": 19 }
   },
+  {
+    "type": "GENERIC",
+    "id": "soaking_spear_shaft",
+    "symbol": "/",
+    "color": "brown",
+    "name": { "str_sp": "soaking spear shaft" },
+    "looks_like": "q_staff",
+    "description": "A long stick carved and sanded into a smooth staff with a slight taper, for use in making spears. It's currently being soaked with oil and quite slippery. Once the oil penetrates the wood it can be wiped down and used for further projects. ",
+    "price": "40 USD",
+    "material": [ "wood" ],
+    "flags": [ "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
+    "weapon_category": [ "QUARTERSTAVES" ],
+    "weight": "1000 g",
+    "volume": "1250 ml",
+    "longest_side": "180 cm",
+    "category": "weapons",
+    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "neutral" },
```

```suggestion
    "to_hit": { "grip": "bad", "length": "long", "surface": "any", "balance": "neutral" },
```
`-1 - "bad" - Particularly hard to grip items, (especially those that are innately slippery or very rounded with no obvious gripping edge) such as basketballs and barrels, or which are dangerous to hold because of very sharp edges, like scrap metal and broken glass.`

~https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/GAME_BALANCE.md